data-structures/stack.md

```diff
@@ -1,24 +1,28 @@
 # Stacks
 
 ### Projected Time
+
 About 1 hour
 
 ### Prerequisites
-JS
-Functional programming
+
+JS Functional programming
```

Comment:
Hi @devRD , sorry to do this, but js and functional prog actually are 2 seperate subjects; could you please change it back?. 
 @Kundan28 , thank you for being conscientious, but prerequisites aren't even in scope since the issue is about materials and supplemental resources. Since a lot of these tickets only regard part of the outline and not the whole thing, please just comment on the issue they are solving, or the changes the contributor actually made. Please note this too, @manaswinidas @hoduchieu01 @saberprashant

Reply:
Okay.